controllers/controller_utils_test.go

```diff
@@ -279,170 +237,6 @@ func expectInitContainer(t *testing.T, podTemplate *corev1.PodTemplateSpec, expN
 	return expInitContainer
 }
 
-// Ensures all the TLS env vars, volume mounts and initContainers are setup for the PodTemplateSpec
```

Comment:
Moved these TLS specific test helper functions to `solrcloud_controller_tls_test` as they are only used by TLS tests and now seem out of place in this file